### ISSUE TITLE
No getters/setters

### ISSUE THREAD

**Issue by bcfox** [enriquein/JavaSetterGetter#6]
I just installed on Sublime Text 2.0.1 and when I try to use this all it does is add an empty line below the member variable I've selected.

I do have vintage mode enabled (not sure if that would cause an issue).

**Comment by enriquein** [comment]
I apologize. This bug was introduced in the last pull request I received. I irresponsibly merged the pull request without testing it because I was under a lot of stress from work and didn't have time to properly test it. I'll do my best to fix this tonight so that you can update tomorrow without issues.

Really sorry for my sloppy managing of this project.

**Comment by bcfox** [comment]
No worries!  Figured you should know.  

I hear you with work.  My own job is driving me nuts right now.  

I'm just glad you seem to have a solution to one of my basic issues with my new favorite IDE.  Right now I have to fire up Eclipse to do large scale object creation and just use ST2 for mods and web stuff.  Looking for plugins to reduce my need to use the old eclipse ball and chain.

I'll update and test it out whenever you have a fix.

**Comment by enriquein** [comment]
By any chance did your member variables happen to be public or generic types?

I checked out the code today and was able to generate getters and setters just fine, except for generic types. Another user provided a fix for that glitch, and that has been fixed in release 1.4.0 which went live 2 minutes ago. 

I ask about the public member variable because the code as it stands will only generate getters/setters for private or protected fields. 

I'm also wondering if it could be an OS issue. What OS are you running Sublime Text 2 on?

**Comment by bcfox** [comment]
The fields are private and I'm on OSX 10.6.8 (can't upgrade OS right now due to other work dependencies).

I updated to 1.4.0 and tested but ran into the same issue.

What I found is that it works as long as you don't define a full package path member variable which I was doing.

For example, the following won't work:
private java.lang.String test = null;

But this will:
private String test = null;

I'll keep this in mind as I use it but I honestly rarely use full package paths (I usually import) this was just a random coincidence that the first one I tried was like this.  It works great otherwise.

Truly, awesome plugin!  Thanks for making ST2 better!
